coremltools/converters/caffe/_caffe_converter.py

```diff
@@ -35,9 +35,13 @@ def convert(model, image_input_names=[], is_bgr=False,
         Arrays) by Core ML.
 
     is_bgr: bool | dict()
-        Flag to determine if the input images are in pixel order (RGB or BGR).
+        Flag indicating the channel order the model internally uses to represent color images.
+        Set to True if the internal channel order is BGR, otherwise it will be assumed RGB.
         Applicable only if image_input_names is specified.
-        To specify different values for each image input provide a dictionary with input names as keys.
+        To specify different values for each image input, provide a dictionary with input names as keys.
+        Note that this flag is about the models internal channel order.
+        An input image can be passed to the model in any color pixel layout containing red, green and blue values (e.g. 32BGRA or 32ARGB).
```

Comment:
This line is a bit long. Long line in docstrings often don't get rendered well. Probably want to fix and fix the other occurrences of this line too.

Reply:
Updated to fit in 80 characters